odl/solvers/functional/default_functionals.py

```diff
@@ -500,6 +503,7 @@ def convex_conj(self):
         [BC2011] Bauschke, H H, and Combettes, P L. *Convex analysis and
         monotone operator theory in Hilbert spaces*. Springer, 2011.
         """
+        # HELP: I guess it needs `range`, too
```

Comment:
Yes, you'll want to use `self.range` (it makes sense IMO, and since it's a `@property` you can't pass arguments).